templates/layout.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    {% block head %}
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
```

Comment:
Depending on our metrics, we may want to move these scripts onto the main page -- but for now a CDN is fine.